src/Application/FileProcessor/PhpFileProcessor.php

```diff
@@ -96,7 +96,7 @@ public function process(File $file, Configuration $configuration): array
 
         // show warning on has InlineHTML node if file has changed
         if ($fileHasChanged && $file->hasInlineHTMLNode()) {
-            $this->symfonyStyle->warning(sprintf(
+            $this->symfonyStyle->getErrorStyle()->warning(sprintf(
```

Comment:
For console format, expected output should show warning, see https://github.com/rectorphp/rector-src/actions/runs/5569673660/jobs/10173190723#step:11:17

you may probably need to have instanceof check for formatter.

Reply:
Sure. But this means the warning only shows up on default cli usage and not when using any other output format. 

IMO stderr in rector is broken, but the instanceof is maybe good enough for now.

Reply:
would it make sense to turn this warning into a system-error?

it looks like its the only warning which is outputted by a worker process this way atm